_includes/markdown/Systems.md

```diff
@@ -0,0 +1,252 @@
+<h2 id="nginx" class="anchor-heading">Nginx{% include Util/top %}</h2>
+
+[Nginx](http://nginx.org/ "Nginx Web Server") is our preferred web server software at 10up. It has proven extremely stable, performant, and reliable at high scale and offers a powerful set of tools. This is not to imply there is anything wrong with using [Apache](https://httpd.apache.org/ "Apache Web Server") - we’ve worked on many high scale deployments that use Apache and mod_php that perform very well.  10up maintains a public set of [Nginx configuration templates](https://github.com/10up/nginx_configs "10up Nginx Configuration Template for WordPress") that apply these best practices. 
```

Comment:
My latest commit adds some text addressing why Nginx over Apache.  It is intentionally weak in the language because Apache can be configured to work as well as Nginx, we just prefer the way Nginx does it.